dspace-spring-rest/src/main/java/org/dspace/app/rest/projection/MetadataOnlyProjection.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.projection;
+
+import org.dspace.content.*;
+import org.dspace.eperson.EPerson;
+import org.springframework.data.rest.core.config.Projection;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Projection interface that only exposes metadata
+ */
+@Projection(name = "metadata",
+        types = { Item.class, Bundle.class, Bitstream.class,
```

Comment:
I would have expected to see the Rest classes (ItemRest.class, BundleRest.class, etc.) here as the projection should define what is included and excluded in the returned Rest objects

Reply:
The projections apply to (Hibernate) entities to prevent the corresponding REST converter to trigger the lazy loading on any unneeded child entities.